src/nilss.jl

```diff
@@ -160,7 +160,7 @@ function NILSSProblem(prob, sensealg::NILSS, t=nothing, dg = nothing; nus = noth
     end
   end
   vstar0 = zeros(eltype(u0), numindvar*numparams)
-  w0 = vec(w[:,:,1,1,:])
+  w0 = vec(w[1,:,1,1,:])
```

Comment:
good catch. Though based on the sentence in the paper: 
"so they could be reused for the new parameter"
it sounds like it should only have an impact on performance.